javascript/javascript1/week2/javascript.js

```diff
@@ -0,0 +1,146 @@
+// Questio :flight booking
+
+function getFullname(firstName,surName,formalName,gender) {
+    if((firstName && surName)==='' ){
```

Comment:
this is not correct because you have an expression that compares a boolean (firstname && surname) and a string...and that is always false
```suggestion
    if(firstName && surName ){
```